src/user/user.service.ts

```
@@ -1,4 +1,36 @@
+
+  async findUser(uid: number) {
+    const useri = await this.prisma.user.findUnique({
+      where: { uid: uid },
+    });
+    if (!useri) {
+      throw new NotFoundException('ID is not found');
+    }
+    return useri;
+  }
+
+  update(id: number, updateUserDto: UpdateUserDto) {
```

ESLint

[error] 'updateUserDto' is defined but never used.

(@typescript-eslint/no-unused-vars)